frontend/src/common-components/AdminHeader.jsx

```diff
@@ -3,18 +3,22 @@ import { Navbar, Nav, Container } from "react-bootstrap";
 
 const AdminHeader = () => {
   return (
-    <Navbar bg="dark" variant="dark" expand="lg">
-      <Container>
-        <Navbar.Brand href="#home">Admin Dashboard</Navbar.Brand>
-        <Navbar.Toggle aria-controls="basic-navbar-nav" />
-        <Navbar.Collapse id="basic-navbar-nav">
-          <Nav className="me-auto">
-            <Nav.Link href="#home">Home</Nav.Link>
-            <Nav.Link href="#link">Settings</Nav.Link>
-          </Nav>
-        </Navbar.Collapse>
-      </Container>
-    </Navbar>
+    <>
+      <Navbar bg="dark" variant="dark" expand="lg" className="admin-header">
+        <Container fluid>
+          <Navbar.Brand href="#home">Admin Dashboard</Navbar.Brand>
+          <Navbar.Toggle aria-controls="basic-navbar-nav" />
+          <Navbar.Collapse id="basic-navbar-nav">
+            <Nav className="ms-auto d-flex d-lg-none">
+              <Nav.Link href="/admin">Home</Nav.Link>
+              <Nav.Link href="/addproducts">Create</Nav.Link>
+              <Nav.Link href="/allorders">All Orders</Nav.Link>
+              <Nav.Link href="/login">Logout</Nav.Link>
```

Comment:
after logging out clear the cookies ref the header component of user page for reference